sklearn/multioutput.py

```diff
@@ -75,13 +75,13 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
 
         Parameters
         ----------
-        X : (sparse) array-like, shape (n_samples, n_features)
+        X : {array-like, sparse matrix} of shape (n_samples, n_features)
             Data.
 
-        y : (sparse) array-like, shape (n_samples, n_outputs)
+        y : {array-like, sparse matrix} of shape (n_samples, n_outputs)
             Multi-output targets.
 
-        classes : list of numpy arrays, shape (n_outputs)
+        classes : list of numpy arrays of shape (n_outputs)
```

Comment:
```suggestion
        classes : list of ndarray of shape (n_outputs,)
```

Reply:
Will correct this and other suggested changes in original PR for multioutput.py #17841